libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -37,4 +41,14 @@ export class SdsFiltersComponent {
   */
   @Input() accordionLabel: string = 'Filters';
 
+    /**
+  *  Emit results when model updated
+  */
+  @Output() filterModel = new EventEmitter<object[]>();
```

Comment:
Do we want a different name for the variable since it is an event/action. Besides that it looks fine.

Reply:
maybe "filterChange"?